packages/CodeGenLib/src/Database/manage-metadata.ts

```diff
@@ -24,6 +24,7 @@ export class ManageMetadataBase {
    public static get newEntityList(): string[] {
       return this._newEntityList;
    }
+   private logFilePath: string = 'MJ/packages/CodeGenLib/src/Database/generated-sql.log';
```

Comment:
@nico-oz  - this isn't good. We need to create a new `section` in the `config.json` file that controls SQL output logging. Lets call that section `SQLOutputLogging` and within that let's have the following variables/options:
* logFilePath: which would be a relative path to the execution of the program, like the other paths in that file
* logBeforeScripts: true or false, default to false, this would result in `before` scripts being logged
* logAfterScripts: true or false, default to false, would result in `after` scripts being logged
* logDynamicSQL: true or false, default to true, would result in the dynamically generated SQL statements being logged - which is what you've done here - but see below as I have comments on how you've implemented this too>

LOGIC: if logFilePath is non-null/non-blank, then each of these 3 boolean values are used to determine if logging is to occur.

Reply:
@cadam11 if you have suggestions on the above please opine here

Reply:
I don't think we should add any of the CodeGen logging parts to `next` just yet. We should ship 2.4 asap, then work on a good solution for creating migration files based on CodeGen afterwards.

We should definitely get log file paths from config, not hard code. Additional controls for optionally logging things that we don't expect to need in migration scripts for the most part isn't a bad idea, but I'd put it in a nice-to-have category.